gamearea.py

```diff
@@ -297,7 +311,7 @@ def __draw_cats(self, context):
 
     def __draw_selected_option(self, context):
         alloc = self.get_allocation()
-        if self.level_data["type"] == GameType.ROWS: 
```

Comment:
There are conflicts coming here when merged with other branches, maybe once all the other pr are reviewed, it will be needed to revisit and then make some changes to resolve them, there is 2 more occurrences of such conflicts.